src/utils/io/progress_log.rs

```diff
@@ -36,6 +36,7 @@
 //! - Add support for bytes measure
```

Comment:
I think you can now remove those doc comments (and we don't want to support ETA).

Reply:
Done! Please re-review.